shared/src/main/scala/coop/rchain/store/KeyValueStoreSyntax.scala

```diff
@@ -12,6 +16,23 @@ final class KeyValueStoreOps[F[_]](
     // KeyValueStore extensions / syntax
     private val store: KeyValueStore[F]
 ) extends AnyVal {
+  // Suffix `1` because overload is not resolved and also because it gets one record
```

Comment:
I am curious.
Does it make sense to make traits work with single key instead of `Seq[key]` and extend it to work with `Seq[Key]` in syntax?

Reply:
It's the easiest way to follow transaction semantics of underlying key-value store. So when multiple keys/pairs are supplied they will be executed atomically.
In the current `Store` this is visible, started with singe record functions it's extended to collections to handle multiple records using one transaction (only for _put_, get is used only for single record).